README.md

```diff
@@ -7,7 +7,7 @@
 </p>
 <p align="center">
     <a href="https://swift.org"><img src="https://img.shields.io/badge/Swift-4.1-orange.svg?style=flat-square"/></a>
-    <a href="https://travis-ci.org/joeblau/gitignore.io"><img src="https://img.shields.io/travis/joeblau/gitignore.io.svg?style=flat-square" alt="Travis"></a>
+    <a href="https://travis-ci.org/toptal/gitignore.io"><img src="https://img.shields.io/travis/toptal/gitignore.io.svg?style=flat-square" alt="Travis"></a>
```

Comment:
src should be: `https://img.shields.io/travis/toptal/gitignore.io/master?style=flat-square` so that it shows the current status for the master branch instead of the last build on travis.

Reply:
Thanks @mcsitter 👍